PS C:\Users\HP\Desktop\BioScriptMedical-1> python .\main.py
MySQL verilənlər bazasına uğurla bağlanıldı
✓ doctors cədvəli mövcuddur
✓ patients cədvəli mövcuddur
✓ prescriptions cədvəli mövcuddur
✓ prescription_items cədvəli mövcuddur
✓ hospitals cədvəli mövcuddur
Bu gün: 0
Bu ay: 0
Ümumi: 0
Pasiyentlər: 0
Traceback (most recent call last):
  File "C:\Users\HP\Desktop\BioScriptMedical-1\main.py", line 204, in on_login_successful
    self.dashboard = BioScriptDashboard(self.db_manager, doctor_data)
                     ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\HP\Desktop\BioScriptMedical-1\ui\dashboard.py", line 348, in __init__
    self.init_ui()
    ~~~~~~~~~~~~^^
  File "C:\Users\HP\Desktop\BioScriptMedical-1\ui\dashboard.py", line 387, in init_ui
    self.ai_assistant = AIAssistantWidget(self.db_manager)
                        ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\HP\Desktop\BioScriptMedical-1\ui\dashboard.py", line 171, in __init__
    self.ai = BioScriptAI()
              ~~~~~~~~~~~^^
  File "C:\Users\HP\Desktop\BioScriptMedical-1\gemini_ai.py", line 16, in __init__
    self.client = genai.Client(api_key=os.environ.get("GEMINI_API_KEY"))
                  ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\HP\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\genai\client.py", line 219, in __init__
    self._api_client = self._get_api_client(
                       ~~~~~~~~~~~~~~~~~~~~^
        vertexai=vertexai,
        ^^^^^^^^^^^^^^^^^^
    ...<5 lines>...
        http_options=http_options,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "C:\Users\HP\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\genai\client.py", line 265, in _get_api_client
    return BaseApiClient(
        vertexai=vertexai,
    ...<4 lines>...
        http_options=http_options,
    )
  File "C:\Users\HP\AppData\Local\Programs\Python\Python313\Lib\site-packages\google\genai\_api_client.py", line 554, in __init__
    raise ValueError(
    ...<3 lines>...
    )
ValueError: Missing key inputs argument! To use the Google AI API, provide (`api_key`) arguments. To use the Google Cloud API, provide (`vertexai`, `project` & `location`) arguments.